pkg/network/payload/address.go

```diff
@@ -6,22 +6,26 @@ import (
 	"time"
 
 	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/network/capability"
 )
 
 // AddressAndTime payload.
 type AddressAndTime struct {
-	Timestamp uint32
-	Services  uint64
-	IP        [16]byte
-	Port      uint16
+	Timestamp    uint32
+	IP           [16]byte
+	Capabilities []capability.Capability
 }
 
 // NewAddressAndTime creates a new AddressAndTime object.
 func NewAddressAndTime(e *net.TCPAddr, t time.Time) *AddressAndTime {
 	aat := AddressAndTime{
 		Timestamp: uint32(t.UTC().Unix()),
-		Services:  1,
-		Port:      uint16(e.Port),
+		Capabilities: []capability.Capability{
```

Comment:
Actually, we should store these along with address in good peers list to properly answer `getaddr`, at the moment we have faked capabilities sent to the other side.